app/views/shared/context_tables/_commodity.html.erb

```diff
@@ -0,0 +1,24 @@
+<table class="govuk-table">
+  <tbody class="govuk-table__body">
+    <tr class="govuk-table__row ">
```

Comment:
I feel like we've all fallen into the trap of treating declarable headings and commodities differently when they should be treated exactly the same. I'm wondering if we should rename this to _declarable.html.erb and just render the declarable or the non-declarable version of this partial.

I'm not picky about it as this is an idiom we're not always using